pyvista/plotting/render_window_interactor.py

```diff
@@ -912,6 +1032,32 @@ def _mouse_left_button_click(self, x=None, y=None, count=1):
             self._mouse_left_button_press(x, y)
             self._mouse_left_button_release()
 
+    def _mouse_middle_button_press(
+        self, x=None, y=None
+    ):  # pragma: no cover # numpydoc ignore=PR01,RT01
+        """Simulate a middle mouse button press.
+
+        If ``x`` and ``y`` are entered then simulates a movement to
+        that position.
+
+        """
+        if x is not None and y is not None:
+            self._mouse_move(x, y)
+        self.interactor.MiddleButtonPressEvent()
+
+    def _mouse_middle_button_release(
+        self, x=None, y=None
+    ):  # pragma: no cover # numpydoc ignore=PR01,RT01
+        """Simulate a middle mouse button release."""
+        if x is not None and y is not None:
+            self._mouse_move(x, y)
+        self.interactor.MiddleButtonReleaseEvent()
+
+    def _mouse_middle_button_click(self, x=None, y=None, count=1):
```

Comment:
```suggestion
    def _mouse_middle_button_click(self, x=None, y=None, count=1):  # pragma: no cover
```

Reply:
I'm okay with this. Do we have a general policy about when to use this?  I'm unsure when we would prefer this over leaving it uncovered so that future PRs can raise coverage.  We shouldn't add this just to see a higher number.

Reply:
I was going to put this part on to make the reader aware that it was not tested, but then I realized that it can be confirmed by going to the codedev page. It is fine as it is.

I would like to make an issue to organize a project-wide policy on skipping coverage.

Reply:
I don't think it is necessarily bad, but it isn't clear to me what it means when we use this in this package.  Some people might expect that it means that there is a reason we aren't testing it.